ds701_book/_quarto-web.yml

```diff
@@ -0,0 +1,95 @@
+project:
+  type: website
+  output-dir: _site
+  pre-render: print_quarto_config.py
+  render:
+    - 01-Intro-to-Python.qmd
+    - 02A-Git-Jupyter.qmd
+    - 02B-Pandas.qmd
+    - 04-Linear-Algebra-Refresher.qmd
+    - 03-Probability-and-Statistics-Refresher.qmd
+    - 05-Distances-Timeseries.qmd
+    - 05A-Learning.qmd
+    - 06-Clustering-I-kmeans.qmd
+    - 07-Clustering-II-in-practice.qmd
+    - 08-Clustering-III-hierarchical.qmd
+    - 09-Clustering-IV-GMM-EM.qmd
+    - 13-Learning-From-Data.qmd
+    - 14-Classification-I-Decision-Trees.qmd
+    - 15-Classification-II-kNN.qmd
+    - 16-Classification-III-NB-SVM.qmd
+    - 10-Low-Rank-and-SVD.qmd
+    - 11-Dimensionality-Reduction-SVD-II.qmd
+    - "!12-Anomaly-Detection-SVD-III.qmd"
+    - 17-Regression-I-Linear.qmd
+    - 18-Regression-II-Logistic.qmd
+    - 19-Regression-III-More-Linear.qmd
+    - "!23-NN-I-Gradient-Descent.qmd"
+    - 24-NN-II-Backprop.qmd
+    - 25-NN-III-CNNs.qmd
+    # For now commenting out 20 because of reliance on 1.2GB train.csv
+    - "!20-Recommender-Systems.qmd"
+    - 21-Networks-I.qmd
+    - 22-Networks-II-Centrality-Clustering.qmd
+
+
+website:
+  title: "Tools for data science"
+  back-to-top-navigation: true
+  sidebar:
+    style: "docked"
+    search: true
+    contents:
+    - href: index.qmd
+      text: Home
+    - section: "Preliminaires"
```

Comment:
There is this typo which was fixed in the previous PR. I think somehow it must have been overwritten.

Reply:
Fixed. Good catch.